plugin/trino-redshift/src/test/java/io/trino/plugin/redshift/TestRedshiftCastPushdown.java

```diff
@@ -491,4 +596,28 @@ protected List<InvalidCastTestCase> invalidCast()
                 new InvalidCastTestCase("c_timetz", "int"),
                 new InvalidCastTestCase("c_timetz", "bigint"));
     }
+
+    @Test
+    void testCastPushdownWithCharConvertedToVarchar()
+    {
+        String table = "t_char_" + randomNameSuffix();
+        // char(REDSHIFT_MAX_CHAR` + 1) is converted to varchar(REDSHIFT_MAX_CHAR` + 1) in Redshift
+        assertUpdate("CREATE TABLE " + table + " (a char(4097))");
```

Comment:
Please use `TestTable` to make sure the table will be deleted in redshift in case of failure.

Reply:
done